Formula/ice.rb

```diff
@@ -12,6 +12,8 @@ class Ice < Formula
   end
 
   option "with-java", "Build Ice for Java and the IceGrid Admin app"
+  option "with-php", "Build Ice for PHP"
+  option "with-python", "Build Ice for Python"
```

Comment:
With

```
option "without-python", "Build without Ice for Python"
depends_on :python => :optional
```

`brew info ice` shows both `--with-python` and `--without-python`... With the workaround `depends_on :python => :optional if build.without?("python")`, only `--without-python` is shown. Is that a good way to do it?